apps/11gist-blog/script.js

```diff
@@ -1,5 +1,72 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+
+    $.ajax('https://api.github.com/users/zekeg/gists', {
+        success: function(response) {
+
+            response.forEach(function(item) {
+
+                //create formatted link item in variable
+                if (item.description.indexOf("#post") >= 0) {
+                    console.log("#post detected!");
+
+                    //create a string with the formatted <a> element and create a new data attribute with the item's url
+                    //add br tag (temp?)
+                    var str = "<li><a href = \"#\" data-url=" + item.url + ">" + item.description.replace("#post", "") + "</a></li>";
+                    //check value in console
+                    console.log(str);
+                    //populate list
+                    $("#posts").append(str);
+                }
+
+                console.log($("body").html());
+
+                $("a[data-url]").click(function(event) {
```

Comment:
Do you find that there are too many of these click handlers being registered? looks like you're creating them in a loop. So you'll add 1, then 2, then 3, and at the end the first element will have 10+ handlers!